### PR TITLE
desktop version

### PR DESCRIPTION
**Pull request by aakbarkhan**
The desktop version was created using the media queries.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @aakbarkhan, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -17,6 +12,134 @@ body {
   align-items: center;
 }
 
+@media screen and (min-width: 768px) {
+  body {
+    width: auto;
+    margin: 0 auto;
+    margin-left: 40px;
+    margin-right: 40px;
+  }
```
Comment (mke2111) [inline style.css:21]: - [ ] For this milestone, the recommended break-point for desktop screen size is between 992px - 1100px. Change your break-point.

Comment (aakbarkhan) [reply style.css:21]: Thank you.

File: index.html [hunk 2]
```diff
@@ -92,100 +171,184 @@ <h6>Skills</h6>
...
+                        </p>
+                        <ul class="languages">
+                            <li>css</li>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>ruby</li>
+                        </ul>
+                        <button class="btn" type="button">See Project</button>
+                    </div>
+                </div>
```
Comment (mke2111) [inline index.html:240]: - [ ] For every project, align the image and button to the bottom. Compare your page Vs the original.
<div class="flex">


</div>

File: style.css (hunk 1 above)
Comment (mke2111) [inline style.css:21]: - [ ] Add the background images to your page.

File: index.html [hunk 3]
```diff
@@ -92,100 +171,184 @@ <h6>Skills</h6>
...
+                            </li>
+                            <li>
+                                <textarea name="text" maxlength="500" id="input" cols="40" rows="10" required
+                                    placeholder="Enter text here"></textarea>
+                            </li>
+                            <li>
+                                <button class="btn-btn" type="submit">Submit</button>
+                            </li>
+                        </ul>
+                    </form>
```
Comment (mke2111) [inline index.html:350]: - [ ] Add all the necessary form fields for the desktop version of your app. Compare your page to the original.

<div class="flex">


</div>
!

Comment (aakbarkhan) [reply index.html:350]: Thank you. 
I am not sure if this was mention in the activity. IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.

Comment (mke2111) [reply index.html:350]: Okay, you may carry on with your design.

Comment (aakbarkhan) [reply index.html:356]: Thank you.

**Review by Leon-Mbegera — APPROVED** [review]
Hi @aakbarkhan , 

# Approved ✔️ 
Well done for addressing all the requested changes 💪 
Your project is complete!  There is nothing else to say other than... it's time to merge it  and proceed to the next stage :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

## Optional
- [ OPTIONAL] This is not a mandatory requirement but guidance since the previous reviewer missed to point it out. The background colors on your social icons in the footer do not match the surrounding background. It would be nice to correct that. Either way, your project is still approved 😄 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**